data/baibhav-tiwari.json

```diff
@@ -3,13 +3,21 @@
   "displayStatsPublic": true,
   "type": "personal",
   "bio": "**Web3** | **Web-developer** | **React** | **Open_Source**",
-
+  "avatar": "https://github.com/baibhav-tiwari.png",
```

Comment:
Delete this avatar line. You don't need to add this. it will automatically fetch.
``` suggestion

```